app/views/specifications/index.html.erb

```diff
@@ -0,0 +1,69 @@
+<%= stylesheet_link_tag 'specifications/index', media: 'all' %> 
+
+<div class='spec-cont'>
+    <div class='specification-header'>
+
+        <%= link_to groups_path do %>
+            <i class="fas fa-arrow-left"></i>
+        <%end%>
+
+        <h1 class='specification_title'>Specifications</h1>
+
+        <%= link_to destroy_user_session_path, data: {turbo_method: :delete, turbo_confirm: "Are you sure?"} do %>
+            <i class="fas fa-sign-out-alt"></i>
+        <% end %>
+
+    </div>
+
+    <div class='specifications-container'>
+        <div class='specification-item'>
+        <img src="<%= @group.icon %>" alt="<%= @group.name %>"> 
+            <span>Category: <%= @group.name %></span>
+
+        </div>
+
+        <div class='specification-item'>
+
+            <p>Total Amount: </p>
+            <span>$<%= @group.total_spent %></span>
+
+        </div>
+
+        <table>
+            <thead>
+                <tr>
+                    <th>Specification</th>
+                    <th>Date</th>
+                    <th>Name</th>
+                    <th>Amount</th>
+                </tr>
+            </thead>
+            <tbody>
+
+                <%if @specifications.empty? %>
+                    <tr>
+                    <td>
+                        <img src="https://img.icons8.com/ios/50/000000/empty-box.png" alt="empty-box">
+                    </td>
```

Comment:
- [ ] Please style this page to look nice on a mobile screen as this project is meant to be a mobile web app. All the styles you did is for desktop but the project is meant to be a mobile web app. Kindly improve your styles as required 👍🏽
- [ ] Kindly reduce the font sizes on mobile screens. They look big and the design doesn't look nice. Please reduce the font sizes 👍🏽


**See how the page looks on a mobile screen**


<img width="462" alt="Screenshot 2023-06-03 at 1 46 55 PM" src="https://github.com/The-Normal-One1/budget_app/assets/50941074/b6a65a2c-2fc7-43fa-9c9c-7ed199336405">